tests/models/video_llava/test_modeling_video_llava.py

```diff
@@ -237,6 +237,9 @@ def test_sdpa_can_compile_dynamic(self):
     def test_sdpa_can_dispatch_on_flash(self):
         pass
 
+    @unittest.skip(
+        reason="After #33533, this still passes, but many subsequential tests fail with `device-side assert triggered`"
+    )
```

Comment:
just a thought, but can it is be because we need to `self.asserts(ValueError)` and not `RuntimeError`. Similar test for video/image_inputs are not causing massive failures

Reply:
I think the desired error (here `RuntimeError`) from such input (`remove some images from inputs`) causing the cdua in a bad state (I tried without assert it). Not sure what would be best approach if we want such test case. But before we have an idea, just skip it 😅

Reply:
oke, thanks! Will have it noted for such tests in the future

Reply:
@zucchini-nlp Running that test in a subprocess could avoid the issue


https://github.com/huggingface/transformers/compare/main...try_sub

But the necessary change is not super trivial